mlflow/java/client/pom.xml

```diff
@@ -85,7 +85,7 @@
         <artifactId>maven-javadoc-plugin</artifactId>
         <configuration>
           <sourcepath>${project.basedir}/src/main/java</sourcepath>
-          <excludePackageNames>com.databricks.api.proto.databricks:org.mlflow.scalapb_interface:org.mlflow.tracking:org.mlflow.artifacts:org.mlflow.api</excludePackageNames>
```

Comment:
Reverts [this change](https://github.com/mlflow/mlflow/pull/2308/files#diff-037c7ca4cfb39d9c9d82de298b20cab0L83), which excludes the tracking API package (`org.mlflow.tracking`) from Javadocs (resulting in empty docs for the MLflow Java client)